substrate/primitives/application-crypto/check-features-variants.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env -S bash -eux
+
```

Comment:
When i try to build a runtime with `app-crypto/full_crypto` i get a missing `#[panic_handler]` error.  
Apparently the `full_crypto` feature also sets `sp-io/disable_panic_handler` and `sp-io/disable_oom`. Is this expected?

Reply:
`full_crypto` won't work (randomness required).